gapii/cc/spy.cpp

```diff
@@ -108,6 +108,9 @@ const uint32_t kStartMidExecutionCapture =  0xdeadbeef;
 const int32_t  kSuspendIndefinitely = -1;
 
 const uint8_t kCoreAPI = 0;
```

Comment:
Agreed. I will add a comment right now.
I haven't figure out a good way to handle this. We could maybe generate some relatively empty header files that we can include without having to deal with weird include cycles.